runtime/lib/registry/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+find_package(pybind11 REQUIRED)
+pybind11_add_module(registry SHARED Registry.cpp)
+target_include_directories(registry PUBLIC ${runtime_includes})
+target_compile_definitions(catalyst_qir_qis_obj PUBLIC -DLIBREGISTRY=\"$<TARGET_FILE_NAME:registry>\")
```

Comment:
We don't want to dynamically link (like -lregistry and setting an rpath). We want to dynamically load, which works at the moment. Otherwise, in the runtime examples (which do not use python at all) we would need `-lregistry`. Any other reason why to set PIC?

Reply:
no! we don't need -fPIC here then :raised_hands: